lms/views/admin.py

```diff
@@ -81,16 +83,14 @@ def update_instance(self):
             "sections_enabled" in self.request.params
             and self.request.params["sections_enabled"] == "on"
         )
+        ai.settings.set("canvas", "sections_enabled", sections_enabled)
+
         groups_enabled = (
             "groups_enabled" in self.request.params
             and self.request.params["groups_enabled"] == "on"
         )
+        ai.settings.set("canvas", "groups_enabled", groups_enabled)
```

Comment:
I couldn't spot any asserts related to these lines in the tests, but it's late and I could just be going cross eyed.

Reply:
Yep, you can delete these lines and the tests pass, which isn't a good sign.